app/konnect/vitals/generate-reports.md

```diff
@@ -1,41 +1,56 @@
 ---
-title: Export Vitals Data into a Report
+title: Export Service and Route Analytics from ServiceHub
 no_version: true
 ---
-
-Browse, filter, and view your {{site.konnect_short_name}} Vitals data in a
-time-series report, and export the data into a comma-separated values (CSV)
-file.
+In the ServiceHub, the Service, Service version, and Route graphs provide dynamic
+graphs with up to 12 hours of data. To view data beyond this time frame, export
```

Comment:
this is correct, but I just want to note that this might change when helium goes out or soon after. Christian may have already talked with you about: https://konghq.atlassian.net/browse/MA-446